### ISSUE TITLE
[menu-button] Initially there are no `aria-details` or `aria-expanded` attributes

### ISSUE THREAD

**Issue by jpzwarte** [sl-design-system/components#1512]
### Provide a general summary of the issue here

Initially there are no `aria-details` or `aria-expanded` attributes, only after you click the button once, those attributes are added.

### 🤔 Expected Behavior?

The attributes should be there initially

### 😯 Current Behavior

-

### 🖥️ Steps to Reproduce

-

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### What operating system are you using?

macOS

### 👤 Your name

Jeroen Zwartepoorte

### 🧢 Your product/team

SLDS

### 🎨 Your Theme(s)

Sanoma Learning
